certbot/certbot/_internal/auth_handler.py

```diff
@@ -265,6 +268,30 @@ def _challenge_factory(self, authzr, path):
 
         return achalls
 
+    def _report_failed_authzrs(self, failed_authzrs):
+        # type: (List[messages.AuthorizationResource]) -> None
```

Comment:
we could just stick this in the method signature now! up to you, I'm fine leaving it just so we can get these in and move on.